mantidproject/autoreduce
# Created pause and resume functionality

Fixes #118

This adds the ability to pause an instrument causing any new runs submitted to be labelled as skipped.

To test WebApp GUI:
* On the 'All Jobs' page confirm that there is a pause/resume instrument link and that clicking it causes the link to change name.
* Resize the 'All Jobs' page to be mobile sized, confirm that the links become icons and operate as expected
* On an instrument summary page confirm that there is a pause/resume button and that pressing it causes expected behaviour

To test functionality:
* Pause an instrument
* Submit a job using a trigger script
* Refresh the 'All Jobs' screen and confirm that the run is shown as skipped and this is not counted as an error
* Resume the instrument and run a job again
* Confirm this run is processed as expected
